scripts/cibuild

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${CI}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Ensure the Terraform source code is properly formatted.
+"
+}
+
+if [[ "${BASH_SOURCE[0]}" == "$0" ]]; then
+    if [[ "${1:-}" == "--help" ]]; then
+        usage
+    else
+        terraform fmt -check .
+        terraform fmt -check ./examples/terraform/
```

Comment:
We talked about this in person, but worth leaving those notes here too. I am of the mind that if you're running `terraform fmt`, it's better to run it recursively without the `--check` flag and auto-commit the new formatting, (I don't have an example of this in public repos, but I can suggest the change it if we go that route.) or include a pre-commit hook that does the formatting and prevents it from getting to the repo. (I use https://github.com/antonbabenko/pre-commit-terraform for this on my repos)